include/pinocchio/multibody/data.hpp

```diff
@@ -505,6 +505,12 @@ namespace pinocchio
     /// \brief Matrix related to potential energy regressor
     RowVectorXs potentialEnergyRegressor;
 
+    /// \brief Matrix related to momentum regressor
+    MatrixXs momentumRegressor;
```

Comment:
Maybe you want to remove `momentumRegressor` and `dpartial_lagrangian_q` or store `Hregressor` and `CTregressor` in them?

Reply:
You are right, I will do this soon!